### ISSUE TITLE
Use option or listitem role for RecursiveList items

### ISSUE THREAD

**Issue by Kyzyl-ool** [gravity-ui/uikit#1346]
"button" role cannot be used as descendant of "listbox"
https://preview.gravity-ui.com/uikit/?path=/story/unstable-uselist-listwithdnd--list-with-dnd

**Comment by amje** [comment]
Fixed
